vrrc.yml

```diff
@@ -3,17 +3,40 @@ channels:
 - conda-forge
 dependencies:
 - python=3.9
-- pandas
-# - boto3
+- boto3
+- botocore
+- cachelib
+- click
 - dash
 - dash-bootstrap-components
 - dash-bootstrap-templates
-- dash-renderer
+- dash-core-components
+- dash-extensions
+- dash-html-components
 - dash-leaflet
+- dash-renderer
+- dash-table
+- EditorConfig
+- Flask
+- Flask-Caching
+- geobuf
+- importlib-metadata
+- itsdangerous
+- Jinja2
+- jmespath
+- jsbeautifier
+- MarkupSafe
+- more-itertools
+- numpy
+- packaging
+- pandas
 - plotly
-# development
-- ipykernel
-- nbformat
-# quality control
-- pylint
-- flake8
+- python-dateutil
+- pytz
+- PyYAML
+- s3transfer
+- six
+- tenacity
+- urllib3
+- Werkzeug
+- zipp
```

Comment:
I really don't believe you need all of these packages. What I had was the minimum requirements, which is what I think we're aiming for. All you have to add to that to make this work is dash-extensions!

Reply:
Fixed

Reply:
Fixed